### PR TITLE
Migrate to AndroidX

### PR DESCRIPTION
**Pull request by rurimo**
- upgrade gradle & gradle tool version
- upgrade comipleSdkVersion(27->28), buildToolsVersion(27.0.3->28.0.3)
- migrate to androidx
- refactor gradle dependencies
- README

### CODE REVIEW

**Comment by wangyung** [comment]
@rurimo thanks for your PR, may I know if you've signed the [ICLA](https://feedback.line.me/enquete/public/2112-yUH6QkxQ)?
Basically LGTM

**Comment by rurimo** [comment]
I signed just now.
Thank you!

File: README.md [hunk 1]
```diff
@@ -15,23 +15,23 @@ Click [here](https://github.com/line/clay/blob/master/screenshot/example.gif) to
...
+```gradle
 allprojects {
     repositories {
         jcenter()
     }
 }
 
 dependencies {
-  compile 'com.linecorp:clayview:0.1.2'
+   implementation "com.linecorp:clayview:0.1.2"
```
Comment (shamenchens) [inline README.md]: is there an extra space? 

```suggestion
  implementation "com.linecorp:clayview:0.1.2"
```

Comment (rurimo) [reply README.md]: Thank you for your review!
I removed the space.

**Comment by wangyung** [comment]
LGTM

**Pull request merged**